health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/AddColumns.js

```
@@ -0,0 +1,126 @@
+            />,
+            <Button
+              type={"button"}
+              size={"large"}
+              variation={"primary"}
+              label={t("ADD")}
+              title={t("ADD")}
+              onClick={() => {
+                addNewCol(`MP_COL_NAME_${colValues.length + 1}`, newColValue)
+                setShowPopUp(false);
+              }}
+            />,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)